bin/run_qc.py

```diff
@@ -1,227 +1,150 @@
+    sp_contamination.add_argument('--fastq_files', required=True, nargs='+', help='fastq file pair')
     sp_contamination.set_defaults(func=run_species_contamination_check)
 
     sample_contamination = subparsers.add_parser('sample_contamination_check')
-    sample_contamination.add_argument('--bam_file', required=True, help='the fastq file pairs')
-    sample_contamination.add_argument('--work_dir', required=False)
-    sample_contamination.add_argument('--sample_id', required=True)
+    sample_contamination.add_argument('--bam_file', required=True)
     sample_contamination.set_defaults(func=run_sample_contamination_check)
 
     gender_val = subparsers.add_parser('gender_validation')
-    gender_val.add_argument('--sample_id', type=str, help='sample ID for creating a Sample dataset object')
     gender_val.add_argument('-v', '--vcf_file', dest='vcf_file', type=str, help='vcf file used to detect gender')
-    gender_val.add_argument('-s', '--working_dir', dest='working_dir', type=str, help='executor work dir')
+    gender_val.set_defaults(func=run_gender_validation)
 
     median_cov = subparsers.add_parser('median_coverage')
     median_cov.add_argument('--bam_file', required=True, help='the fastq file pairs')
-    median_cov.add_argument('--work_dir', required=False)
-    median_cov.add_argument('--sample_id', required=True)
     median_cov.set_defaults(func=median_coverage)
 
     contam_blast = subparsers.add_parser('contamination_blast')
     contam_blast.add_argument('--fastq_file', required=True, nargs='+', help='fastq file to check for contamination')
-    contam_blast.add_argument('--sample_id', required=True)
     contam_blast.set_defaults(func=contamination_blast)
 
     relatedness_parser = subparsers.add_parser('relatedness')
-    relatedness_parser.add_argument('--gVCF_files', required=True, nargs='+')
+    relatedness_parser.add_argument('--gvcfs', required=True, nargs='+')
     relatedness_parser.add_argument('--reference', required=True)
-    relatedness_parser.add_argument('--work_dir', required=False)
-    relatedness_parser.add_argument('--project_id', required=True)
     relatedness_parser.set_defaults(func=relatedness)
 
     return parser.parse_args()
 
 
-def run_genotype_validation(args):
-    def retrieve_data(paths, work_dir, allow_fail=False):
-        cmd = rsync_from_to(paths, work_dir)
-        exit_status = executor.execute(
-            cmd, job_name='retrieve_data', working_dir=work_dir, cpus=1, mem=2
-        ).join()
-        if exit_status != 0 and not allow_fail:
-            raise AnalysisDriverError("Copy of the file(s) from remote has failed")
-
+def run_genotype_validation(dataset, args):
     # Get the sample specific config
     cfg.merge(cfg['sample'])
-    projects_source = cfg.query('output_dir')
-    work_dir = os.path.join(cfg['jobs_dir'], args.sample_id)
-    os.makedirs(work_dir, exist_ok=True)
-
-    # Hack to retrieve the fastq file from the CIFS share
```

Comment:
Removed remote file rsyncing, since as far as I know, we haven't needed it for a while.